core/src/main/java/org/apache/carbondata/core/scan/expression/ExpressionResult.java

```diff
@@ -183,7 +183,7 @@ public String getString() throws FilterIllegalMemberException {
             return parser.format((java.sql.Date) value);
           } else if (value instanceof Long) {
             if (isLiteral) {
-              return parser.format(new Timestamp((long) value / 1000));
+              return parser.format(new Timestamp((long) value));
```

Comment:
@chenliang613 earlier time stamp was getting multiplied by 1000 in carbondata so thats why it was wriiten as 
-return parser.format(new Timestamp((long) value / 1000));
but now it is not getting multipled by 1000 so at time of building expression result so now there is no need of dividing timestamp by 1000